### ISSUE TITLE
Can't require spire.selmer from any other namespace

### ISSUE THREAD

**Issue by josesanch** [epiccastle/spire#64]
spire -e "(require '[spire.selmer])"
Exception in thread "main" clojure.lang.ExceptionInfo: Could not require spire.selmer. [at line 1, column 1] {:type :sci/error, :line 1, :column 1, :message "Could not require spire.selmer. [at line 1, column 1]"}
	at sci.impl.utils$rethrow_with_location_of_node.invokeStatic(utils.cljc:74)
	at sci.impl.utils$rethrow_with_location_of_node.invoke(utils.cljc:58)
	at sci.impl.interpreter$eval_call.invokeStatic(interpreter.cljc:522)
	at sci.impl.interpreter$eval_call.invoke(interpreter.cljc:502)
	at sci.impl.interpreter$interpret.invokeStatic(interpreter.cljc:551)
	at sci.impl.interpreter$interpret.invoke(interpreter.cljc:534)
	at sci.impl.interpreter$eval_form.invokeStatic(interpreter.cljc:592)
	at sci.impl.interpreter$eval_form.invoke(interpreter.cljc:583)
	at sci.impl.interpreter$eval_string_STAR_.invokeStatic(interpreter.cljc:603)
	at sci.impl.interpreter$eval_string_STAR_.invoke(interpreter.cljc:597)
	at sci.core$eval_string_STAR_.invokeStatic(core.cljc:193)
	at sci.core$eval_string_STAR_.invoke(core.cljc:189)
	at spire.eval$evaluate.invokeStatic(eval.clj:88)
	at spire.eval$evaluate.invoke(eval.clj:86)
	at spire.core$_main.invokeStatic(core.clj:73)
	at spire.core$_main.doInvoke(core.clj:57)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at spire.core.main(Unknown Source)
Caused by: java.lang.Exception: Could not require spire.selmer.
	at sci.impl.interpreter$handle_require_libspec.invokeStatic(interpreter.cljc:236)
	at sci.impl.interpreter$handle_require_libspec.invoke(interpreter.cljc:206)
	at sci.impl.interpreter$eval_require$fn__4283.invoke(interpreter.cljc:264)
	at clojure.core$run_BANG_$fn__8795.invoke(core.clj:7717)
	at clojure.lang.PersistentVector.reduce(PersistentVector.java:343)
	at clojure.core$reduce.invokeStatic(core.clj:6829)
	at clojure.core$run_BANG_.invokeStatic(core.clj:7712)
	at clojure.core$run_BANG_.invoke(core.clj:7712)
	at sci.impl.interpreter$eval_require.invokeStatic(interpreter.cljc:264)
	at sci.impl.interpreter$eval_require.doInvoke(interpreter.cljc:240)
	at clojure.lang.RestFn.applyTo(RestFn.java:139)
	at clojure.core$apply.invokeStatic(core.clj:667)
	at clojure.core$apply.invoke(core.clj:660)
	at sci.impl.namespaces$require.invokeStatic(namespaces.cljc:448)
	at sci.impl.namespaces$require.doInvoke(namespaces.cljc:447)
	at clojure.lang.RestFn.applyTo(RestFn.java:139)
	at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
	at clojure.lang.RestFn.invoke(RestFn.java:421)
	at clojure.core$partial$fn__5843.invoke(core.clj:2626)
	at sci.impl.interpreter$fn_call.invokeStatic(interpreter.cljc:467)
	at sci.impl.interpreter$fn_call.invoke(interpreter.cljc:467)
	at sci.impl.interpreter$eval_call.invokeStatic(interpreter.cljc:518)

**Comment by retrogradeorbit** [comment]
That's a bit of an oversight! You can access it from `user` namespace or `spire.modules` namespace as a work around for the missing namespace.

```shell
 spire -e "user/selmer"
#<Fn@5d2eec16 spire.selmer/selmer>
$ spire -e "spire.modules/selmer"
#<Fn@5d2eec16 spire.selmer/selmer>
```

**Comment by retrogradeorbit** [comment]
fixed in 0.1.0-alpha.13
